File: typescript/05-is-divisible.ts
// ¿Es divisible?
// Fácil
// 10 pts
// Números

// Enunciado
// Dados dos números n y divisor, devuelve true si n es divisible por divisor (sin residuo).

// Ejemplo
// isDivisible(10, 2)  → true
// isDivisible(7, 2)   → false
// isDivisible(9, 3)   → true
// Puedes usar console.log() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.

function isDivisible(value: number, divisor: number): boolean {
  return value % divisor == 0;
}

// No modificar: necesario para evaluar el resultado.
export { isDivisible };

File: python/05_is_divisible.py
"""
¿Es divisible?
Fácil
10 pts
Números

Enunciado
Dados dos números n y divisor, devuelve true si n es divisible por divisor (sin residuo).

Ejemplo
is_divisible(10, 2)  # True
is_divisible(7, 2)   # False
is_divisible(9, 3)   # True
Puedes usar print() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.
"""


def is_divisible(value: int, divisor: int) -> bool:
    return value % divisor == 0
